projects/demo-maps/src/app/route-components/route-example-olperformance/fetchbutton/fetchbutton.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { LayersService, VectorLayer } from '@dlr-eoc/services-layers';
+import { Fill as olFill, Stroke as olStroke, Style as olStyle } from 'ol/style';
+import simplify from '@turf/simplify';
+import bbox from '@turf/bbox';
+import { reproject } from 'reproject';
+import proj4 from 'proj4';
+import { BehaviorSubject } from 'rxjs';
+
+
+type fetchState = 'ready' | 'fetching' | 'done' | 'error';
+
+
+@Component({
+  selector: 'app-fetchbutton',
+  templateUrl: './fetchbutton.component.html',
+  styleUrls: ['./fetchbutton.component.scss']
+})
+export class FetchbuttonComponent implements OnInit {
+
+  public state$ = new BehaviorSubject<fetchState>('ready');
+
+  constructor(
+    private http: HttpClient,
+    private layersSvc: LayersService
+  ) { }
+
+  ngOnInit(): void {
+  }
+
+  fetchClicked() {
+    const fileUrl = 'https://bloomington.in.gov/geoserver/publicgis/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=publicgis:Buildings&outputFormat=application%2Fjson';
+    this.state$.next('fetching');
+    this.http.get(fileUrl).subscribe((data) => {
+      this.addLayers(data);
+      this.state$.next('done');
+    });
+  }
+
+  private addLayers(dataOrig: any) {
+
+    const str2966 = '+proj=tmerc +lat_0=37.5 +lon_0=-87.08333333333333 +k=0.999966667 +x_0=900000 +y_0=249999.9998983998 +datum=NAD83 +units=us-ft +no_defs';
+    const data = reproject(dataOrig, str2966, proj4.WGS84);
```

Comment:
can you try to use openLayers for this as well [Reprojecting geometries in vector layers](https://openlayersbook.github.io/ch07-wrapping-our-heads-around-projections/example-04.html)